From a7a71e73bfb56ba21a25d8d32fca7e062ce23afe Mon Sep 17 00:00:00 2001
From: Ksawlii <ksawery.blaszczak@proton.me>
Date: Fri, 21 Mar 2025 19:16:16 +0100
Subject: [PATCH] Run hyprland with dbus

Signed-off-by: Ksawlii <ksawery.blaszczak@proton.me>
---
 hyprland.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hyprland.desktop b/hyprland.desktop
index bb2801a..9a7f2f4 100644
--- a/hyprland.desktop
+++ b/hyprland.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=Hyprland
 Comment=An intelligent dynamic tiling Wayland compositor
-Exec=Hyprland
+Exec=dbus-run-session Hyprland
 Type=Application
 DesktopNames=Hyprland
 Keywords=tiling;wayland;compositor;
-- 
2.48.1

